.github/workflows/pr-realm-js.yml

```diff
@@ -294,8 +296,15 @@ jobs:
       - name: Install pods
         if: ${{ matrix.variant.environment == 'react-native' && matrix.variant.os == 'ios'}}
         working-directory: integration-tests/environments/${{ matrix.variant.environment }}
+        env:
+          USE_HERMES: ${{ matrix.variant.engine == 'hermes' && '1' || '0' }}
         run: npx pod-install
 
+      - name: Set hermesEnabled
+        working-directory: integration-tests/environments/${{ matrix.variant.environment }}
+        if: ${{ matrix.variant.os == 'android' }}
+        run: echo "hermesEnabled=${{ matrix.variant.engine == 'hermes' }}\n" >> android/gradle.properties
```

Comment:
I don't think this is correct anymore - this is how we do it in the install tests: https://github.com/realm/realm-js/blob/main/install-tests/react-native/patches/jsc-build.gradle.patch .. or perhaps we can use a `GRADLE_OPTS` environment variable.